sphinx/source/docs/user_guide/server.rst

```diff
@@ -1,41 +1,44 @@
 .. _userguide_server:
 
-Running a Bokeh Server
+Running a Bokeh server
 ======================
 
 .. _userguide_server_purpose:
 
 Purpose
 -------
 
-The purpose of the Bokeh server is to make it easy for Python users to create
-interactive web applications that can connect front-end UI events to real,
-running Python code.
-
-The architecture of Bokeh is such that high-level "model objects"
-(representing things like plots, ranges, axes, glyphs, etc.) are created
-in Python and then converted to a JSON format that is consumed by the
-client library, BokehJS. (See :ref:`userguide_concepts` for a more detailed
-discussion.) By itself, this flexible and decoupled design offers advantages.
-For instance, it is easy to have other languages (R, Scala, Lua, ...) drive
-the exact same Bokeh plots and visualizations in the browser.
-
-However, if it were possible to keep the "model objects" in Python and in
-the browser in sync with one another, more additional and powerful
-possibilities immediately open up:
-
-* respond to UI and tool events generated in a browser with computations or
-  queries using the full power of Python
-* automatically push server-side updates to the UI (i.e. widgets or plots in a browser)
+.. note::
+    To make this guide easier to follow, consider familiarizing
+    yourself with some of the core concepts of Bokeh in the section
+    :ref:`userguide_concepts`.
+
+Bokeh server makes it easy to create interactive web applications that connect
+front-end UI events to running Python code.
+
+Bokeh creates high-level Python models, such as plots, ranges, axes, and
+glyphs, and then converts these objects to JSON to pass them to its client
+library, BokehJS. For more information on the latter, see
+:ref:`devguide_bokehjs`.
+
+This flexible and decoupled design offers some advantages. For instance, it is
+easy to have other languages, such as R, Scala, or Lua, drive Bokeh plots and
```

Comment:
Let's nix Lua from this list. The others are in various disrepair but Lua never really happened to begin with.